packages/asc-ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import { FacebookPadded } from '@datapunt/asc-assets'
+import Accordion from './Accordion'
+import Paragraph from '../Paragraph'
+import AccordionWrapper from './AccordionWrapper'
+import Icon from '../Icon'
+
+export default {
+  title: 'Experimental/Atoms/Accordion',
+
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div style={{ padding: '40px 10px' }}>{storyFn()}</div>
+    ),
+  ],
+}
+
+export const SingleAccordionWithParagraph = () => (
+  <Accordion title="Only a paragraph" id="one">
+    <Paragraph gutterBottom={0}>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad delectus hic
+      incidunt iure labore natus odio pariatur qui sint, voluptas!
+    </Paragraph>
+  </Accordion>
+)
+
+export const SingleAccordionWithCustomContent = () => (
+  <Accordion title="With custom content" id="one">
+    <Icon padding={4} size={48}>
+      <FacebookPadded />
+    </Icon>
+  </Accordion>
+)
+
+export const SingleAccordionWithAHugeTitle = () => (
+  <Accordion
+    title="Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci aperiam consequatur id, ipsum, libero minima praesentium qui recusandae repellat, repellendus reprehenderit vel? Amet beatae consequatur debitis, ex, fuga hic id itaque laborum neque odio officia optio perferendis tempore tenetur totam."
+    id="one"
+  >
+    <Paragraph gutterBottom={0}>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad delectus hic
+      incidunt iure labore natus odio pariatur qui sint, voluptas!
+    </Paragraph>
+  </Accordion>
+)
+
+export const SingleAccordionWithAControlledState = () => {
+  const [open, setOpen] = useState(false)
+
+  setTimeout(() => {
+    setOpen(true)
+  }, 3000)
```

Comment:
Although for this is probably fine for in this case, we should probably set an example and use the proper hooks with cleanup here.

```suggestion
useEffect(() => {
  const timer = setTimeout(() => setOpen(true), 3000)
  return () => clearTimeout(timer)
}, [])
```

Reply:
good one!